nl/_drafts/subsurface-4-9-4-released.MD

```diff
@@ -1,7 +1,7 @@
 ---
 layout: post
-title:  Aankondiging Subsurface 4.9.4
-lang:   nl
+title: "Aankondiging Subsurface 4.9.4"
+lang: "nl"
```

Comment:
I believe (??) the quotes are optional, but I can't claim to be an expert by any stretch of the imagination.
They definitely don't hurt.